packages/shared/src/components/ShareBar.tsx

```diff
@@ -42,53 +53,155 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
         </SimpleTooltip>
-        <SimpleTooltip content="Share on Twitter">
-          <ColorfulShareButton
+
+        <SimpleTooltip content="Share on Facebook">
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
             tag="a"
-            href={getTwitterShareLink(href, post.title)}
+            href={getFacebookShareLink(href)}
             target="_blank"
             rel="noopener"
-            onClick={() => onClick(ShareProvider.Twitter)}
-            icon={<TwitterIcon secondary />}
-            className="btn-tertiary"
-          />
+            onClick={() => onClick(ShareProvider.Facebook)}
+            textPosition="justify-start"
+            icon={
+              <div className="flex justify-center items-center mb-2 group-hover:bg-theme-hover rounded-xl">
+                <FacebookIcon secondary className="w-7 h-7 text-white" />
+              </div>
+            }
+          >
+            Facebook
+          </Button>
         </SimpleTooltip>
-        <SimpleTooltip content="Share on Facebook">
-          <ColorfulShareButton
+
+        <SimpleTooltip content="Share on Twitter">
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
             tag="a"
-            href={getFacebookShareLink(href)}
+            href={getTwitterShareLink(href, post.title)}
             target="_blank"
             rel="noopener"
-            onClick={() => onClick(ShareProvider.Facebook)}
-            icon={<FacebookIcon secondary />}
-            className="btn-tertiary"
-          />
+            onClick={() => onClick(ShareProvider.Twitter)}
+            textPosition="justify-start"
+            icon={
+              <div className="flex justify-center items-center mb-2 group-hover:bg-theme-hover rounded-xl">
+                <TwitterIcon secondary className="w-7 h-7 text-white" />
+              </div>
+            }
+          >
+            Twitter
+          </Button>
         </SimpleTooltip>
+
+        {squads
+          ?.filter(
+            (squadItem) =>
+              squadItem.active &&
+              verifyPermission(squadItem, SourcePermissions.Post),
+          )
+          ?.map((squad) => (
+            <SimpleTooltip content={`Share on ${squad.name}`} key={squad.id}>
+              <Button
+                className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
+                key={squad.id}
+                onClick={() => onShareToSquad(squad)}
+                textPosition="justify-between"
+                icon={
+                  <div className="flex justify-center items-center mb-2">
+                    <SourceProfilePicture
+                      source={squad}
+                      className="w-10 h-10 squad"
```

Comment:
I think for this we also support sizing, secondary, why is the squad class needed?